amgut/handlers/auth_handlers.py

```diff
@@ -42,7 +42,7 @@ def post(self):
             ag_login_id, skid, kitinfo['password'],
```

Comment:
Great, thanks!

On Mon, Feb 23, 2015 at 5:39 PM, Joshua Shorenstein <
notifications@github.com> wrote:

> In amgut/handlers/auth_handlers.py
> <https://github.com/biocore/american-gut-web/pull/373#discussion_r25218242>
> :
>
> > @@ -42,7 +42,7 @@ def post(self):
> >              ag_login_id, skid, kitinfo['password'],
>
> I think that would work nicely, since it catches any issues with passwords
> not being migrated silently for the user.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/373/files#r25218242>.
>